tests/TestsBase.php

```diff
@@ -23,6 +23,8 @@ public function setUp()
 
         parent::setUp();
 
+        App::make('config')->set('translatable.translation_model_namespace', 'Dimsav\Translatable\Test\Model');
```

Comment:
I know that it's in a lot of places in unittests but can you please switch to `$this->app->make('config')` instead of the Facade.